### DISCUSSION TITLE
Obico support

### DISCUSSION THREAD

**Discussion by Zylvo** [mainsail-crew/spyglass#38] [Q&A]
Hello,
is anybody using spyglass with obico?
Obico is reporting that:
"moonraker-obico: Webcam Streaming Failed
The webcam streaming failed to start. Obico is now streaming at 0.1 FPS."
and I wonder if that issue is only on my setup or Obico is not working well with spyglass

**Comment by roamingthings** [comment]
I haven't heard from Obico yet. What setting did you use?

**Reply by Caouette1988** [reply]
For me, spyglass work on 32bit with Obico. Had to manually install mainsail, moonraker and klipper. 
I install them with kiauh. 

The mainsailOS was not working. 

64bit was not working too.

**Reply by Zylvo** [reply]
Actually, it's working now, I don't know what solved it, cause I've made some network changes yesterday/today. I had this problem since beginning with spyglass and it was still there 3 days ago, so when I had time today, I've posted to ask. Now I've started print and it's working. I'm using 64bit so it shouldn't work, but it's working. Very strange ¯\_(ツ)_/¯

Only thing that comes to my mind is that either last spyglass or moonraker-obico updates fixed it or my network configuration changes (which is less likely)
